core/src/main/scala/KubernetesClient.scala

```diff
@@ -135,6 +135,25 @@ object KubernetesClient {
 }
 
 object KubernetesJson {
+  val DEFAULT_CPU = 0.5
```

Comment:
do we use uppercase constants elsewhere in the nelson codebase? don't think we do - this looks a little funny to my eye :-)

Reply:
fixed!